### ISSUE TITLE
Klee Wyck: Opening Sketches comparison: 1941 to 1951

### ISSUE THREAD

**Issue by ebeshero** [newtfire/introDH-Hub#23]
First, here's a **[simple view](https://docs.google.com/document/d/1rfk4FIyjxUXHsEwavE9ArLN_GpSgBt1DlvRmmcX7R6o/edit?usp=sharing)** of the original 1941 text of the first three sketches in Emily Carr’s *Klee Wyck*: 

Now, here's a Google Docs **[comparison view](https://docs.google.com/document/d/1MkRY7FoYb4NQk-CerQdYi2GT7ZF3DRo7BojoAvxc1Zk/edit?usp=sharing)** of the first three sketches in Emily Carr’s *Klee Wyck*, setting 1941 as the "base text" and tracking changes applied in the 1951 edition: 

Problems we need to read around = changes that are not really changes: formatting, spacing. 

What alterations stand out to you? How did the text change between 1941 and 1951? And what do you think the significance of those changes are? Point out some alterations (you can use screen captures, and/or describe and quote from some altered passages), and comment on a particular kind of alteration you are seeing. How does this alteration change the way you read the passage, or the sketch as a whole?

**Comment by arrowarchive** [comment]
I have a lot to say about this. 

First and foremost, the majority of the changes I have witnessed involve spacing, spelling, and punctuation. In one paragraph, the word "school-girl" is corrected to "school girl", removing the hyphen. Another instance of changes in punctuation is on page five, where the line "but I found that a grunt of welcome was always waiting inside" was edited in the 1951 version to remove "that" from the sentence. Similarly, on the same page "Papooses tumbled round her" rephrased it to "tumbled around" in 1951. These changes are minor, but the more advanced changes come later.

The missionaries and the church have a lot of information cut for syndication, with several instances of the word "Presbyterian" and dialogue concerning the snootiness of the lesser missionary being cut, making her into a flat character compared to the other missionary. The 1951 version makes the missionaries look less antagonistic, cutting their dialog that demeans the Indians after they tell the origin of Klee Wyck, the scene with them praying in the bedroom, and how indignant they were when the Indians spoke to each other in their native language instead of English. Though a number of the lesser Missionary's dialog is cut, the insults spoken by the greater missionary are also cut. 

The most notable instance of censorship, in my opinion, was how the rituals of the dead were cut from the 1951 version. I understand the argument that it could be considered gruesome or disturbing, but it was a part of their culture. In order to understand a culture, a person can't just pick and choose what aspects they want to show, and this was an example of this.

Overall, the 1951 version makes the missionaries appear more heroic, but it makes their characterizations flatter. The lesser missionary doesn't have a lot of defining traits other than her gender and having a daughter, and the greater missionary is no better. Their characterizations, along with the rituals surrounding the dead, are cuts to the story I found the most interesting.

**Comment by am0eba-byte** [comment]
Of the many cuts and alterations that the 1951 edition made on the original 1941 version, the deletion of this last paragraph in Ucluelet was the most disappointing to me: 

"Up behind Toxis the forest climbed a steep hill and here in the woods
was one lonely grave, that of 'our only professed Christian Indian',
according to the Missionaries. The Missionaries had coffined him tight
and carried him up the new-made trail with great difficulty. They put
him into the earth among the roots of the trees, away from all his
people, away from the rain and the sun and the wind which he had loved
and which would have rushed to help his body to melt quickly into the
dust to make earth richer because this man had lived."

The total removal and censorship of the last few paragraphs in Ucluelet where Klee Wyck described the Native American tradition of handling their dead was indeed saddening overall, but I felt that the removal of this final powerful passage is what made it truly disrespectful to the novel as a whole. It is so moving and revealing - it's a truly significant passage, especially because it tells the tale of what happens when you rip a person from their cultural roots. This passage is hardly gruesome or gory at all, and yet the editors of the 1951 edition chose to cut this beautiful passage from the story entirely. They could've just edited some of the "gory" words in the passages prior to it to make it palpable for school kids, but they really just ripped the soul out of the whole sketch (and arguably the heart of the story) by taking out that section.

A small point I'd like to make on the notion of keeping things PG for the kids, by way of cutting out "gore" or "gruesome" passages - It seems they felt fine about cutting those passages describing the Native American tradition of handling their dead, and yet they left this sentence in Tanoo: 

"Do you remember what they said about those Indians being asphyxiated by
by the fumes from their engine while they slept?”

Did that one slip past the editors? Granted, I don't think that sentence is very gory, but the passages they cut at the end of Ucluelet are *hardly* any more gruesome than this. In fact, I actually thought they were less gruesome, but I guess that's just an opinion.


There was another pretty decent bit that was removed from the Tanoo sketch in the 1941 version that I couldn't quite wrap my head around:

![mosquitos_m mission_KW](https://user-images.githubusercontent.com/70165410/98989561-4d757f00-24f7-11eb-9359-9607a2eee3c3.png)

I could understand how one might deem the first paragraph "sensitive," on account of the dominating tone Carr puts on the Mission house in relation to the tribe's villiage, but the rest of that passage and the next one that was cut do not seem very religiously/culturally sensitive to me. There are a lot more instances after this bit where the parts containing the missionary girl's arrogance (and also a lot of parts adding to the complexity of her character) are cut as well, and seemingly for no reason. Even sentences as innocent as "The chatter of the missionary's daughter in solemn Skedans sounded like a sheep bell tinkling outside a church," were cut, therein cutting out a huge chunk of the dynamics of her character.

**Comment by BenS6206** [comment]
There were many areas where there were only minor revisions that mostly revolved around correcting grammar and such, but where I found more interesting changes was in the large areas that were removed or altered for instance this section talking about the stupidity of the missionary girl, and some lines of the daughter were also removed. 
![image](https://user-images.githubusercontent.com/69773455/99060305-d29b7b00-256d-11eb-9b40-bdb536e4886a.png)
Obviously this was in place to curb the complaints about the missionaries in general, as it may have been deemed offensive or sensitive. Overall a lot of important details were cut throughout the sketches, trying to sanitize it of anything that could be viewed as sensitive, which frankly diminishes the work through oversimplification and removal of the authors intent. It changes how the entire story is interpreted as a whole.

**Comment by xsierraallen** [comment]
When comparing Emily Carr's 1941 text to her 1951 revision, I noticed some changes that stood out more than others.
![Screenshot (36)](https://user-images.githubusercontent.com/70916885/99088389-fc6a9700-2599-11eb-8d6d-c863071af3d9.png)
This section of the text in 1951 revision, the paragraph describing the missionaries was rewritten. I am not exactly sure why, but they were described differently the second time. The 1941 text ay have been too lengthy when describing these characters in my opinion. This however, does not make much of an impact when comparing both texts.
![Screenshot (37)](https://user-images.githubusercontent.com/70916885/99088974-b8c45d00-259a-11eb-9939-0ec9543b4601.png)
In this image, you are able to see the multiple changes in the text. It appears to me that most of these revisions are a word being crossed out only for it to be rewritten the same way. I'm not sure if I am missing something but these changes are not even changes at all. Usually these deal with spelling errors and punctuation marks but I cannot see the difference.
To be honest, I feel as if the 1951 version did a better job at pulling out wordy messes and getting to the point. Clearly describing something without taking up too much space. After all, that is the job of a revision of a text right? Not many big changes were made, only small ones here and there. Although the biggest change I was able to find was this:
![Screenshot (38)](https://user-images.githubusercontent.com/70916885/99089479-5e77cc00-259b-11eb-8cfa-268fad9b06c9.png)
In some places, chunks of text were taken out and thrown away. They may have been too wordy or maybe just not needed to get the story across. However, this happens multiple times throughout the text. Cutting these details out may be good for some people, but bad for others. These details were obviously important in 1941 so who is to say they do not matter in 1951? This sort of thing is what really changes the story as a whole.

**Comment by dxh405** [comment]
Looking past all of the changes made to the grammar and spacing, and spelling changes you will find a lot of things cut that were almost more critical of the missionaries. While these cuts tend to provide a much quicker explanations in some revisions I feel they are leaving out details that are taking away the character of some of them. There were several spots where the 51 version omitted things that would bring a different light to a character or several characters. For example there is an omission that describes the one missionary girl as clumsy and stupid. 
![clumsy](https://user-images.githubusercontent.com/70723878/99102397-05646400-25ac-11eb-9d35-a4c303f7f835.PNG)
Along with this there are some things describing some processes that the missionaries do that were taken out. Overall a lot of the bigger things that were taken out were things directly about missionaries and their character. This kind of thing really does change how people view them and changes a meaning as a whole.

**Comment by pxl5083** [comment]
Looking through the text, a few things are apparent, at least to me. The first thing that I noticed was that a lot of sections regarding missionaries were censored or cut out, to go even further than that, they were not only things about missionaries but negative things or descriptions of the missionaries. The thing about this, is that when people change descriptors about characters, it tends to change the way people end up viewing the alleged character group. Now that being said, I imagine that the censorship taking place was for that exact purpose, nonetheless it changes the archetype and way the characters are perceived. This in-turn changes the narrative of the story, which is undoubtedly a disappointment.

Also sorry for the lack of pictures I will add them later, at the moment they are not working properly.

**Comment by kzp308** [comment]
When I compared the two versions, I analyzed Ucluelet and Tanoo. I noticed that there were definitely a lot of changes in the 1951 version. However, I realize that most of the corrections have to do with spelling or punctuation errors. For example, in the first sentence there was a change in spelling where the author had changed "missionaries" to "Missionaries". It's a small change, but while reading the sentence the capitalized M makes more sense to me. The author continued to make tiny changes throughout the text, but I did notice she took out a small paragraph in the original version.  
![Screenshot (100)](https://user-images.githubusercontent.com/70867065/99108691-bae7e500-25b5-11eb-97d9-0263658f0996.png)
In the screenshot, it shows the author took out a paragraph that was in the 1941 version. In the 1951 version, it was removed and replaced with just "summer".

**Comment by tomsheehy** [comment]
While analyzing the Ucluelet, I realized that the passages that were removed seemed to be removed because paragraphs were too wordy. The changes made the passage easier to understand. The picture below shows a good example as a full paragraph was taken out and replaced with "had the most dignity".
<img width="1158" alt="Screen Shot 2020-11-13 at 1 43 05 PM" src="https://user-images.githubusercontent.com/70911575/99109447-d7d0e800-25b6-11eb-89ae-92d29dc666e4.png">
In the Tanoo version, we also see an example of this. In this example, we see that "and the missionary girl's sneer" was removed. Because of this, there is less detail and information and it makes the passage more simple, with the same message.

**Comment by bpm5520** [comment]
Analyzing the Tanoo portion of the sketches, it is noticeable that a lot of the edits were of the same word being crossed out and rewritten the same. I am not sure why that is so, but it seemed to take up a significant portion of the edits. 
![image](https://user-images.githubusercontent.com/70908691/99111571-6430da00-25ba-11eb-9159-c7408476f72a.png)
Other edits seemed to be of shortening or deleting sentences from the paragraph, which helped to make the sections less wordy and easier to read through. These edits helped keep the story going smoothly without adding too much detail, and I think improved the flow of the work.
![image](https://user-images.githubusercontent.com/70908691/99111798-ba9e1880-25ba-11eb-947b-7d8585381923.png)
It is also interesting that there were 2 whole paragraphs taken out of the piece. These 2 paragraphs focused on describing the Indians and the mosquito bites on everyone, and I'm glad it was removed, as it seemed like way too much detail for the scene that is taking place.
![image](https://user-images.githubusercontent.com/70908691/99111988-fb962d00-25ba-11eb-9fb3-10b0e47c6e31.png)

**Comment by jzm6677** [comment]
when comparing 1941 to 1951 I'd notice that a lot of paragraphs and sentence where just taken out.
![passage](https://user-images.githubusercontent.com/70913490/99113112-b672fa80-25bc-11eb-835f-b99a837006d5.png)
Going farther it was interesting a lot of it was the same just some words got crossed out. It was weird due to the fact words would get crossed off but be re written. As well as it had a lot of grammar correction it seemed or at least it was slightly reworded. 
![passage2](https://user-images.githubusercontent.com/70913490/99113407-34cf9c80-25bd-11eb-917d-bfb7782cf7ef.png)

**Comment by JesseBeckwith1998** [comment]
During my comparison of the 1941 and the 1951 texts, I noticed that most of the differences are of smaller stature in terms of them being spacing and capitalization differences.
however, there are some larger differences between the two texts with an example on the very first page of text where after I did a comparative analysis of the two documents through google, there is roughly half a paragraph removed signifying a significant difference between them.
That being so, there are not many more examples of multiple sentences being crossed out. As I had mentioned in the first sentence, the reoccurring theme of differences lies in capitalization and spacing, there are dozens of spacing differences in just the first 3.5 pages that I used comparison analysis on. 
I will a screenshot or two to show some examples in space differential. 
![Gutenberg_comparison pic 1](https://user-images.githubusercontent.com/70914661/99113962-10c08b00-25be-11eb-90b4-1638e80f8738.png)
![Gutenberg_comparison pic 2](https://user-images.githubusercontent.com/70914661/99113970-14541200-25be-11eb-845c-3bbaae0e2f95.png)
As you can see in the above images, there are 2 examples of a sentence or more being changed between the two texts while spacing is the primary difference.

**Comment by amw6765** [comment]
While looking for the changes that were made in the 1951 edition, I came across a lot of small changes that had been made. They were usually a word here and there that had been taken out or a string of words in sentences that had been taken out. Most of the time you were still able to understand what was being said regardless if they had been altered in a small way. Also, I came across instances where some words in the 1941 edition that were lowercase but made to be capitalized in the 1951 revision.

<img width="611" alt="Screen Shot 2020-11-13 at 2 54 12 PM" src="https://user-images.githubusercontent.com/70722523/99116033-60ed1c80-25c1-11eb-91ae-a9a28427f2a2.png">

However, as I was furthermore looking through the text I was able to see that a lot of paragraphs had been removed in the 1951's revision. This had seemed to occur quite often and with it happening so much it had shortened the the length of the original immensely. As I read what was removed and after remembering what we had discussed in class, it seems that the sections removed were passages that may be seen as offensive or inappropriate. Due to this being taught in classrooms it may not have been what was needed to remain in order for it to be appropriately taught to students. Here are some of the biggest edits that I came across that were removed in the 1951 revision:

<img width="535" alt="Screen Shot 2020-11-13 at 2 54 21 PM" src="https://user-images.githubusercontent.com/70722523/99116050-66e2fd80-25c1-11eb-8332-629eecdef87a.png">

<img width="530" alt="Screen Shot 2020-11-13 at 2 54 41 PM" src="https://user-images.githubusercontent.com/70722523/99116070-6c404800-25c1-11eb-857b-ff390483b4fa.png">

<img width="331" alt="Screen Shot 2020-11-13 at 2 55 36 PM" src="https://user-images.githubusercontent.com/70722523/99116081-706c6580-25c1-11eb-8133-3ad1ab6b0759.png">

**Comment by nxh5137** [comment]
While reading, I noticed that there were a lot of paragraph breaks, whether this is because of the Google Doc or the actual text I wouldn't know. Other than that though I have noticed some changes to the text. For example, on the eighth page, an entire paragraph is deleted from the story. This paragraph talked about what the missionaries were doing in the bedroom. So in the 1951 version, "The room was deathly still." is what comes immediately after instead of that paragraph, as can be seen in the screenshot below.
![Screenshot 2020-11-13 at 1 13 53 PM](https://user-images.githubusercontent.com/34407859/99110626-afe28400-25b8-11eb-8e24-9ea5d0ed4fc7.png)

Another way something was deleted from the original story can be seen below. In the 1951 version, only a lesser missionary speaks while the other one is censored after what they say.
![Screenshot 2020-11-13 at 1 30 38 PM](https://user-images.githubusercontent.com/34407859/99115218-0ef7c700-25c0-11eb-8de9-a6ccc546f8e6.png)

One final thing I noticed is that something does get censored majorly about halfway through the Google Doc. It seems to discuss a place that is surrounded by death, giving us some disturbing imagery of human remains as well as talking about Indians. The screenshot below is not big enough to show the entirety of what was censored, but this goes on to the next page and then the next chapter begins.
![Screenshot 2020-11-13 at 2 55 25 PM](https://user-images.githubusercontent.com/34407859/99115937-3dc26d00-25c1-11eb-96a8-5d1d497c9bd0.png)

As can be seen by these screenshots, anything was not safe from being censored, whether it was a sentence or an entire page, the story from 1951 is really different from the original.

**Comment by Joelpie** [comment]
There are a vast number of changes between the two versions of text, and they range in size from omitting a hyphen or comma all the way to omitting entire paragraphs of description. The aforementioned grammatical corrections (though some can be called mere adjustments, as some words are moved to the next line without being grammatically incorrect) did not alter my reading experience, and instead providing more context into the evolution of writing conventions.

![image](https://user-images.githubusercontent.com/60297307/99117526-dc4fcd80-25c3-11eb-9603-ddc5c006336d.png)
Here's an example of some words being moved to the next line.

In contrast to these minor corrections, the omitting of large portions of descriptive text definitely altered my reading experience, in a number of ways. The first of these is the censorship of imagery. Censorship and I haven't always got along, in fact during my junior year of highschool I wrote a speech on why we should just do away with the idea altogether. Though this was an extreme take from 16 year-old me, I don't completely disagree with it even today. Take this excerpt for example.

![image](https://user-images.githubusercontent.com/60297307/99117783-56805200-25c4-11eb-9a7f-cd0891ce2454.png)

I've excluded the latter half, as this portion is enough to get the message across. These three paragraphs are a grim, eerie, downright disturbing description of the burial rituals conducted for the "Indian dead". I believe this is an important piece to leave in the entire text, as the imagery is staggering and wraps itself in a veil of seriousness that could otherwise be broken.

All that being said, I can definitely understand why this is censored. This next excerpt is a bit beyond me though.

![image](https://user-images.githubusercontent.com/60297307/99118156-035acf00-25c5-11eb-8c94-6003e6049e8e.png)

It's almost as if the very concept of undergarments was deemed too scandalous and left out of the printing, which is a bit silly to me. Maybe the target demographic was changing, and in response these changes were, or vice versa.

**Comment by mws6176** [comment]
Reading over both editions, the common error to occur seemed to be the lack or capitalization. There were several different spots where this happened in each version. Here is just one example of a failure to capitalize: 
![Screen Shot 2020-11-13 at 3 33 17 PM](https://user-images.githubusercontent.com/70854997/99118669-f2f72400-25c5-11eb-87e1-1062bf0ac639.png)

**Comment by Janman813** [comment]
In this comparison to both of these versions, I noticed that the grammar had changed in different ways. Some of the ways are adding hyphens, commas, new capitalization. For example, on the first page first sentence you see missionaries in lower case in the older version and then in the newer version, it is capitalized. another thing is on the first page and in she wrote "a fifteen year old school girl", but changed it to " fifteen-year-old school girl.

**Comment by hjl5363** [comment]
For this assignment, I really love how Dr. Bondar was able to use a Google Doc to visually show comparisons. I think this was much better than the Variorum Viewer. The dots and highlights that were constantly on Variorum Viewer were confusing, and I think a Google Doc is better in the long run, especially in the remote environment that we are currently in. More specifically, I like how Google Docs is more specific and detailed in regards to the changes that are made. The more detail there is, the better. 

<img width="950" alt="Screen Shot 2020-11-13 at 3 43 08 PM" src="https://user-images.githubusercontent.com/70916576/99119638-83823400-25c7-11eb-9fea-dcc019a9a744.png">

In terms of noticeable changes, I would say that a big trend was the constant change in grammar between the two versions. This ranged from a variety of things such as structure in a paragraph and adding/ changing of punctuation, such as changing “Missionary’s” to "missionary's”.

<img width="1400" alt="Screen Shot 2020-11-13 at 4 09 42 PM" src="https://user-images.githubusercontent.com/70916576/99121627-baa61480-25ca-11eb-9f8b-95ef31b2267a.png">

Another big trend that could be noticed throughout the comparison of the two versions is the complete deletion of long part of the story to simplify into a couple of words. I personally think that this was done to try to get rid of unnecessary content that was not needed to comprehend the story and to keep the reader engaged. I think this is a strategy that was very well implemented.

<img width="694" alt="Screen Shot 2020-11-13 at 4 03 33 PM" src="https://user-images.githubusercontent.com/70916576/99121182-2176fe00-25ca-11eb-93f3-218062cee53b.png">
<img width="1231" alt="Screen Shot 2020-11-13 at 4 03 16 PM" src="https://user-images.githubusercontent.com/70916576/99121183-220f9480-25ca-11eb-9e8a-489fd08cdc5e.png">

In addition, there were several times where thoughts were revised, but at the same time not deleted in an effort to make the thoughts of the author more clear to the reader. An example of this is when “made my curiosity and the missionary girl’s sneer” was replaced with “made my curiosity”.

**Comment by gabbiedoster** [comment]
When analyzing the differences between then 1941 version and 1951 of Emily Carr's _Kyee Wyck_, obvious trends I found were differences in punctuation and grammar. Although, I find that to be a commonly seen trend in revised pieces of work. The most interesting finds I found between both works was the amount of large paragraphs that were eventually removed in the later 1951 edition of _Kyle _Wyck_. I found these most interesting because some if not most of these paragraphs were  extremely detailed and personally I thought held onto major development in the story.  In the image below is an example of a screen capture I've decided to analyze. These two paragraphs hold common meaning and develop the stories characters such as Miss Missionary and Louisa. Keeping these paragraphs in the 1951 edition would've been a great opportunity to further develop what these characters attributes are. 
<img width="767" alt="Screen Shot 2020-11-17 at 9 54 42 PM" src="https://user-images.githubusercontent.com/70911505/99477792-3aace100-2921-11eb-8e33-9572582fc83c.png">
